src/mwoffliner.lib.ts

```diff
@@ -75,8 +75,8 @@ async function execute(argv: any) {
     keepEmptyParagraphs,
     mwUrl,
     mwWikiPath,
-    mwActionApiPath,
     mwApiPath,
```

Comment:
The `mwApiPath` is ambiguous, now that MW provides many API end-points. Please rename `mwApiPath` to `mwActionApiPath` everywhere (not only on the command line).

Reply:
Done.